app/meldingen/incident/containers/IncidentContainer/saga.js

```diff
@@ -1,6 +1,49 @@
-// import { take, call, put, select } from 'redux-saga/effects';
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+import request from 'utils/request';
+
+import { CREATE_INCIDENT, GET_CLASSIFICATION } from './constants';
+import {
+  createIncidentSuccess,
+  createIncidentError,
+  getClassificationSuccess,
+  getClassificationError
+} from './actions';
+import makeSelectIncidentContainer from './selectors';
+
+export function* createIncident() {
+  const requestURL = '/api/meldingen';
+
+  try {
+    const data = yield select(makeSelectIncidentContainer());
+    console.log('data', data);
+    const incident = yield call(request, requestURL);
+    yield put(createIncidentSuccess(incident));
+  } catch (err) {
+    yield put(createIncidentError(err));
+  }
+}
+
+export function* getClassification({ text }) {
+  const requestURL = 'http://meldingen-classification.herokuapp.com/calls/';
```

Comment:
use this through the dyson middleware.
const requestURL="<extern-api-name>/getclasification";

in proxy.conf.dev.js you can add the entry:
'/<extern-api-name>/*': {
      target: 'http://meldingen-classification.herokuapp.com/calls/',
      pathRewrite: { '^/<extern-api-name>/': '' },
      secure: false,
      changeOrigin: true,
      logLevel: 'debug'
    }